bin/flashair.rpi

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright (c) 2016 Stephen Warren <swarren@wwwdotorg.org>
+# Copyright (c) 2016 Konsulko Group
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# * Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+# * Neither the name of the copyright holder nor the names of its contributors
+#   may be used to endorse or promote products derived from this software
+#   without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+flashair_rmlist=`mktemp`
+
+case "${board_type}" in
+rpi)
+    kernel_dst=kernel.img
+    ;;
+rpi_2)
+    kernel_dst=kernel7.img
+    ;;
+rpi_3)
+    kernel_dst=kernel8.img
+    ;;
+rpi_3_32b)
+    kernel_dst=kernel8-32.img
+    ;;
+*)
+    echo Unknown Pi \""${board_type}"\"
+    exit 1
+    ;;
+esac
+
+echo 'kern*.img' > "${flashair_rmlist}"
+
+cp "${U_BOOT_BUILD_DIR}"/u-boot.bin "${UL_DIR}"/"${kernel_dst}"
+
+touch "${UL_DIR}/config.txt"
```

Comment:
Shouldn't this rm config.txt first, or 'echo > config.txt' to make sure it's zero-length? Otherwise, the append below will keep making it larger every time.